src/domain/shared/vo/email.go

```diff
@@ -21,7 +22,7 @@ func NewEmail(email string) (Email, error) {
 func isValidEmail(email string) error {
 	rxEmail := regexp.MustCompile(emailPolicy)
 	if !rxEmail.MatchString(email) {
-		return fmt.Errorf("無効なEmailアドレスです")
+		return apperr.BadRequest("無効なEmailアドレスです")
```

Comment:
```suggestion
		return apperr.BadRequestf("無効なEmailアドレスです: %s", email)
```

後にデバッグしやすいようにその時のメールアドレスを入れておくと良いです。

他のエラーハンドリングも、後でデバッグしやすいようにしておくと良いです！

Reply:
修正しました。